Mathlib/Order/Filter/AtTopBot/Field.lean

```diff
@@ -42,21 +42,21 @@ lemma tendsto_div_const_atTop_of_pos (hr : 0 < r) :
   simpa only [div_eq_mul_inv] using tendsto_mul_const_atTop_of_pos (inv_pos.2 hr)
 
 /-- If `f` tends to infinity along a nontrivial filter `l`, then
-`fun x ↦ r * f x` tends to infinity if and only if `0 < r. `-/
+`fun x ↦ r * f x` tends to infinity if and only if `0 < r. ` -/
```

Comment:
Here and in the other comments below: please also remove the trailing `. Your fix is good, but not complete :-)

Reply:
Done!